src/bindings/python/src/openvino/utils/postponed_constant.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from typing import Callable, List
+import openvino
+
+"""Postponed Constant is a way to materialize a big constant only when it is going to be serialized to IR and then immediately dispose."""
+
+
+# `maker` is a function that returns ov.Tensor that represents a target Constant
+def make_postponed_constant(element_type: openvino.Type, shape: openvino.Shape, maker: Callable[[], openvino.Tensor]) -> openvino.Op:
+    class PostponedConstant(openvino.Op):
+        class_type_info = openvino.DiscreteTypeInfo("PostponedConstant", "extension")
+
+        def __init__(self) -> None:
```

Comment:
A regular constant has a name. I believe that `PostponedConstant` should have a name too.

Reply:
done